.github/workflows/ci-backend.yml

```diff
@@ -1,8 +1,9 @@
-name: test_backend
+name: backend
 
 on:
   push:
     branches:
+      - master
```

Comment:
I don't get it. It means one won't be able to run CI on a feature branch unless it merged to master

Reply:
Unless you open a pr, push to master or merge to master.
Now it runs on every push (when PR is opened it runs twice because of push + pull_request)

Reply:
but this is exactly my flow. I do a new branch and expect pushed commits to run in CI. Postponing it till the merge request will significantly increase the feedback loop in this case

Reply:
rolledback